src/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -129,6 +129,11 @@ protected function doTestFiles(array $files): void
         $this->autoloadTestFixture = true;
     }
 
+    protected function doTestFile(string $file): void
+    {
+        $this->doTestFiles([$file]);
```

Comment:
For now i solved it like this, so it can be refactored later, but can already be used in tests with more readable api.

Reply:
Good enough